packages/main/src/plugin/container-registry.ts

```diff
@@ -891,6 +902,21 @@ export class ContainerProviderRegistry {
     return matchingContainerProvider.api;
   }
 
+  protected getMatchingContainerProvider(
```

Comment:
I thought this method should use code from getMatchingEngineFromConnection and getMatchingEngineFromConnection should call this new method and just return .api no ?

Reply:
done 👍